101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/GettingStarted.md

```diff
@@ -0,0 +1,150 @@
+# Deploy a Load Balancer, Network Security Group, a Virtual Network and an Azure Databricks Workspace with the Virtual Network
+
+The purpose of this ARM Template is **Deploy a Load Balancer, Network Security Group, a Virtual Network and an Azure Databricks Workspace with the Virtual Network** inserting a few parameters.
+
+## The Template
+
+Don't let the size of the template scares you. The structure is very intuitive and once that you get the gist of it, you will see how much easier your life will be deploying resources to Azure.
+
+These are the parameters on the template, most of them already have values inserted, the ones that you need to inform are: **adminUsername**, **adminPassword** and **resourceGroup**.
+
+Parameter         | Suggested value     | Description
+:--------------- | :-------------      |:---------------------
+**WorkspaceName** |  | The name of your DataBricks Workspace.
+**Pricing Tier** | 14 Day Trial, Standard or Premium
+**Resource Group** The Resource Group which you wish to deploy your DataBricks Environment. 
+**Disable Public Ip** | Default is false | Set this to true to disable Public IP address
+
+All the other parameters can be left as default.
+
+**Nsg Name** Name of the network Security Group
+**Vnet Name** Name of the Virtual Network
+**Private Subnet Name** Name of the Private Subnet
+**Public Subnet Name** Name of the Public Subnet
+**Location** Location of Data Center
+**Vnet Cidr** Cidr Range of the Vnet
+**Private Subnet Cidr** Cidr Range of the Private Subnet
+**Public Subnet Cidr** Cidr Range of the Public Subnet
+**Load Balancer Backend Pool Name** Name of the Backend Pool of the Load Balancer
+**Load Balancer Frontend Config Name** Name of the Frontend Load Balancer configuration
+**Load Balancer Name** Name of the Load Balancer
+**Load Balancer Public IP Name** Name of the outbount Load Balancer Public IP
+
+## Deployment
```

Comment:
remove deployment section and more the rest to the readme.md

Reply:
done